### PR TITLE
CONT-990_Adding PR template

### PR DESCRIPTION
**Pull request by malikparvez**
CONT-990_Adding PR template

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Note : we are trying to define a template for PR, any suggestions are welcome, get in touch with CAT team...
+
+## Summary
+Provide a detailed description of the changes in this pull request.
```
Comment (jordanbreen28) [inline .github/pull_request_template.md:2]:
```suggestion
Provide a detailed description of all the changes present in this pull request.
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+## Additional Context
+Add any other context about the problem here.
```
Comment (jordanbreen28) [inline .github/pull_request_template.md:5]:
```suggestion 
Add any additional context about the problem here. (Root cause/steps to reproduce (if applicable), thought process on implementation)
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+## Related Issues (if any)
+Mention any related issues or pull requests.
+
+## Checklist
+
+- [ ] Rspec.
```
Comment (jordanbreen28) [inline .github/pull_request_template.md:12]:
```suggestion
- [ ] 🟢 Spec tests.
```

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+- [ ] Acceptance tests.
```
Comment (jordanbreen28) [inline .github/pull_request_template.md:13]:
```suggestion
- [ ] 🟢 Acceptance tests.
```

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+- [ ] Manually tested.
```
Comment (jordanbreen28) [inline .github/pull_request_template.md:14]:
```suggestion
- [ ] Manually verified. (For example `puppet apply`)
```

File: .github/pull_request_template.md (hunk 1 above)
Comment (malikparvez) [reply .github/pull_request_template.md]: changed

File: .github/pull_request_template.md (hunk 2 above)
Comment (malikparvez) [reply .github/pull_request_template.md]: changed

File: .github/pull_request_template.md (hunk 3 above)
Comment (malikparvez) [reply .github/pull_request_template.md]: done

File: .github/pull_request_template.md (hunk 4 above)
Comment (malikparvez) [reply .github/pull_request_template.md]: done

File: .github/pull_request_template.md (hunk 5 above)
Comment (malikparvez) [reply .github/pull_request_template.md]: done

File: .github/pull_request_template.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
+## Summary
+Provide a detailed description of all the changes present in this pull request.
+
+## Additional Context
+Add any additional context about the problem here. (Root cause/steps to reproduce (if applicable), thought process on implementation)
```
Comment (jordanbreen28) [inline .github/pull_request_template.md:5]:
```suggestion
Add any additional context about the problem here. 
- [ ] Root cause and the steps to reproduce. (If applicable)
- [ ] Thought process on the implementation.

```

**Review by jordanbreen28 — DISMISSED** [review]

**Review by jordanbreen28 — APPROVED** [review]

**Pull request merged**